src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckConfig.java

```diff
@@ -1,15 +1,77 @@
 package au.org.emii.gogoduck.worker;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Enumeration;
+import java.util.Properties;
+
 public class GoGoDuckConfig {
```

Comment:
This looks like it used to be a defaults/constants class - this is probably misnamed.  I'd keep your layerconfig separate.